 * QA Notice: Package has poor programming practices which may compile
 *            fine but exhibit random runtime failures.
 * 	tabe_tsi.c:385: warning: incompatible implicit declaration of built-in function ‘fabs’
 *	tabe_tsi.c:437: warning: incompatible implicit declaration of built-in function ‘fabs’
 *	tabe_tsi.c:496: warning: incompatible implicit declaration of built-in function ‘fabs’
 * Please do not file a Gentoo bug and instead report the above QA
 * issues directly to the upstream developers of this software.
 * Homepage: https://packages.qa.debian.org/libt/libtabe.html

diff -NrU5 libtabe-0.2.6.orig.really/src/tabe_tsi.c libtabe-0.2.6.orig/src/tabe_tsi.c
--- libtabe-0.2.6.orig.really/src/tabe_tsi.c	2008-04-20 01:35:56.000000000 +0200
+++ libtabe-0.2.6.orig/src/tabe_tsi.c	2008-04-20 01:36:46.000000000 +0200
@@ -12,10 +12,11 @@
 #endif
 
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <math.h>
 
 #include "tabe.h"
 
 struct TsiInfo *
 tabeTsiInfoNew(char *str)
